j2lint/linter/error.py

```diff
@@ -21,24 +23,22 @@ def __repr__(self, verbose=False):
             error = json.dumps({"id": self.rule.id,
                                 "message": self.message,
                                 "filename": self.filename,
-                                "linenumber": self.linenumber,
+                                "line_number": self.line_number,
                                 "line": self.line,
                                 "severity": self.rule.severity
                                 })
         else:
             if not settings.verbose:
-                formatstr = u"{2}:{3} {5} ({6})"
+                formatstr = "{2}:{3} {5} ({6})"
             else:
-                formatstr = u"Linting rule: {0}\nRule description: {1}\nError line: {2}:{3} {4}\nError message: {5}\n"
+                formatstr = "Linting rule: {0}\nRule description: " \
+                            "{1}\nError line: {2}:{3} {4}\nError message: {5}\n"
```

Comment:
Do not use `\` to break line - cf PEP8: https://peps.python.org/pep-0008/#maximum-line-length
```suggestion
                formatstr = ("Linting rule: {0}\nRule description: "
                             "{1}\nError line: {2}:{3} {4}\nError message: {5}\n")
```